### PR TITLE
parity-version pr reopen

### PR DESCRIPTION
**Pull request by debris**

### CODE REVIEW

**Comment by 5chdn** [comment]
tests fail

**Comment by debris** [comment]
yes, cause linking C++ libraries is somehow broken and behaves in unpredictable way 😕

**Comment by arkpar** [comment]
Needs a rebase. Snappy issues should be fixed in master now.

File: Cargo.toml [hunk 1]
```diff
@@ -94,13 +95,12 @@ ui-precompiled = [
 ui-enabled = ["dapps"]
 dapps = ["parity-dapps"]
 jit = ["ethcore/jit"]
-dev = ["clippy", "ethcore/dev", "ethcore-util/dev", "ethsync/dev", "parity-rpc/dev", "parity-dapps/dev"]
```
Comment (tomusdrw) [inline Cargo.toml]: Why is `ethcore-util/dev` removed from here?

Comment (debris) [reply Cargo.toml]: `ethcore-util/dev` does not exist any more

Comment (tomusdrw) [reply Cargo.toml]: Fair enough, anyway see #7355

**Pull request merged**